tools/synchro.py

```diff
@@ -89,6 +90,8 @@
 
 type_replacements = {"_Condition": "threading.Condition"}
 
+import_replacements = {"test.synchronous": "test"}
```

Comment:
This is for the conversion process, due to the bulk of our tests still using the original `test/__init__.py` and associated helpers. Once we've fully migrated all tests into the `test/asynchronous` and `test/synchronous` directories, we will switch to using `test.synchronous.<module>` again.